### PR TITLE
Add Beta 2r2t to V7

### PR DESCRIPTION
**Pull request by RebornTrack970**
Other Omega Servers will likely get updated to V7 too, for now, only 2r2t has a Beta.

### CODE REVIEW

File: servers_v7.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 [
   {
     "address": "mindustry.us.to"
+  },
+  {
+    "name": "Omega"
```
Comment (Quezler) [inline servers_v7.json:6]:
```suggestion
    "name": "Omega",
```

Comment (Quezler) [reply servers_v7.json]: also, this same ip is also (still) on the bleeding edge list 🤔

Comment (RebornTrack970) [reply servers_v7.json]: k thx

Comment (RebornTrack970) [reply servers_v7.json]: yeah, gona remove it from there too

**Comment by RebornTrack970** [comment]
Now fully finished.

**Review by Quezler — APPROVED** [review]

**Pull request merged**